tempesta_fw/str.h

```diff
@@ -210,7 +213,7 @@ tfw_str_eolen(const TfwStr *s)
  * Updates EOL length value
  */
 static inline void
-tfw_str_set_eolen(TfwStr *s, unsigned int eolen)
+tfw_str_fixup_eolen(TfwStr *s, unsigned int eolen)
```

Comment:
The change is incorrect. There are two different functions that do different things. One is `tfw_str_set_eolen()`, and the other is `tfw_str_fixup_eol()`. Please read the code and the comments for both functions.